buildmaster/ATLAS_Z0_8TEV_HIGHMASS/metadata.yaml

```diff
@@ -0,0 +1,56 @@
+setname: "ATLAS_Z0_8TEV_HIGHMASS"
+
+version: 1
+version_comment: "First implementation"
+
+# References
+arXiv:
+  url: "https://arxiv.org/abs/1606.01736"
+iNSPIRE:
+  url: "https://inspirehep.net/literature/1467454"
+hepdata:
+  url: "https://www.hepdata.net/record/ins1467454"
+  version: 1
+
+nnpdf_metadata:
+  nnpdf31_process: "DY NC"
+  experiment: "ATLAS"
+
+implemented_observables:
+
+- observable_name: "2D"
+  observable:
+    description: "Measurement of the double-differential cross section for the Drell-Yan at 8 TeV"
+    label: r"$d\sigma / dM d|y|$"
+    units: "[fb]"
+  process_type: "EWK_RAP"
+  tables: [2]
+  ndata: 48
+  npoints: [48]
+  plotting:
+    kinematics_override: ewk_rap_sqrt_scale
+    dataset_label: "ATLAS DY 2D 8 TeV low mass"
```

Comment:
This is the high mass dataset, is the rest of the plotting data equal between the two?